Felienne/hedy
# Adds norwegian to supported languages

**Description**
In this PR we add Norwegian as a supported language to our web structure. We also fix a small issue where Chinese wouldn't work with the new Babel structure as we used the `zh` abbreviation whereas Babel expected `zh_hans`. This should all work now.

**Fixes**
This PR fixes #2327 

**How to test**
Navigate to the home page and verify that Norwegian (Norsk) is now an option on the language selector. Also verify that this is working as expected. **Make sure to compile Babel first before checking**.
